google/cloud/bigtable/data/_async/mutations_batcher.py

```diff
@@ -203,25 +204,18 @@ def __init__(
           - flow_control_max_bytes: Maximum number of inflight bytes.
           - batch_operation_timeout: timeout for each mutate_rows operation, in seconds. If None,
               table default_operation_timeout will be used
-          - batch_per_request_timeout: timeout for each individual request, in seconds. If None,
-              table default_per_request_timeout will be used
+          - batch_attempt_timeout: timeout for each individual request, in seconds. If None,
+              table default_attempt_timeout will be used
         """
         self._operation_timeout: float = (
-            batch_operation_timeout or table.default_operation_timeout
+            batch_operation_timeout or table.default_mutate_rows_operation_timeout
         )
-        self._per_request_timeout: float = (
-            batch_per_request_timeout
-            or table.default_per_request_timeout
+        self._attempt_timeout: float = (
+            batch_attempt_timeout
+            or table.default_mutate_rows_attempt_timeout
             or self._operation_timeout
```

Comment:
Should this be tables default_mutate_rows_operation_timeout? Because that's used as the default operation_timeout

Reply:
You mean instead of self._operation_timeout?

My intent was to set the attempt time as the full operation time if it's not set anywhere else. I used self._operation_timeout instead of the table default because the operation timeout may have been overridden by the batch_operation_timeout, and if so we should use that.

Or am I thinking about this wrong?

Reply:
I dont think mutate_rows/read_rows should default to the unary timeouts

Reply:
`self._operation_timeout` isn't the unary timeout here, it is set as `self._operation_timeout = batch_operation_timeout or table.default_mutate_rows_operation_timeout`. 

So the attempt timeout is falling back to the operation_timeout detected for this call, either passed in directly or from the table default